package.json

```diff
@@ -10,8 +10,7 @@
     "micro-services"
   ],
   "dependencies": {
-    "@hyperswarm/dht": "0.1.0",
-    "async": "^2.6.1",
+    "@hyperswarm/dht": "git://github.com/hyperswarm/dht.git#propagate-update-errors",
```

Comment:
note: temporarily depending on this branch until https://github.com/hyperswarm/dht/pull/26 is reviewed and merged